emigrate nobles (#1382)
DFHack/scripts: changelog.txt

## New Features
- `advtools`: new ``advtools.fastcombat`` overlay (enabled by default) allows you to skip combat animations and the announcement "More" button by mashing the movement keys
- `gui/journal`: now working in adventure mode

> - `emigration`: new command for sending barons of other sites back to the sites that they rule over

Reviewer: ```suggestion
- `emigration`: new ``nobles`` command for sending "freeloader" barons back to the sites that they rule over
```
